### DISCUSSION TITLE
Add support for astral UV as python dependency manager

### DISCUSSION THREAD

**Discussion by sebglon** [aquasecurity/trivy#7647] [Ideas]
### Description

the new project [Astrav UV ](https://github.com/astral-sh/uv) is becoming the benchmark for python dependency management.
Can you at it to dependency scan?

### Target

None

### Scanner

None

**Comment by knqyf263** [comment]
Thanks for your suggestion. It makes sense. Created https://github.com/aquasecurity/trivy/issues/7653
